src/components/Login.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import auth from "../modules/auth";
+
+const Login = props => {
+  const [displayLoginButton, setDisplayLoginButton] = useState(true);
+  const [loginMessage, setLoginMessage] = useState("");
+
+  const onLogin = event => {
+    event.preventDefault();
+    auth
+      .signIn(event.target.email.value, event.target.password.value)
+      .then(userDatas => {
+        props.changeAuth(true);
+        setLoginMessage(`Logged in as: ${userDatas.data.email}`);
+      })
+      .catch(error => {
+        alert("Invalid login credentials");
+        window.location.reload();
```

Comment:
Not sure if we should have to reload the page like this.  Not at all

Reply:
Removed it.

Reply:
Good, from a user perspective I dislike having to re-enter my email/username after using the wrong password